### PR TITLE
Update addon to work with ember-cli 0.2.7

### PR DESCRIPTION
**Pull request by irnc**
I need to use typeahead with remote dataset, for this there needs to be an ability to extent base component which now works only with local dataset.

This is the first step in which code was adjusted to run with latest ember-cli and expose component to consuming application, so it can be extended.

Later pull request would change component internals to allow extensions.

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
```
Comment (irnc) [inline LICENSE.md]: This addon does not have a license. Could you please add one, to allow this pull request?

MIT was the default from `ember init`.

**Comment by irnc** [comment]
Build failed for ember-beta. Could we fix it in separate pull request to keep changes in this as small as possible?

**Comment by Robdel12** [comment]
I updated the deps and the tests passed for me so I merged. Here's a question for you @irnc: would you like to take this repo over and become the maintainer?

**Comment by irnc** [comment]
Thanks! I just noticed that my changes reseted version in package.json to 0.0.0 - I will fix it tomorrow.

I will work with this component for the future weeks, and plan to integrate Bloodhound and ember-data more tightly, if needed and possible, and describe recipes on using typeahead.js with Ember. Following this plan I can be a contributor, and if all goes well I will be happy to also maintain that component later.

**Pull request merged**